content/develop/_index.md

```diff
@@ -10,35 +10,89 @@ tags = [ "theme" ]
+
+## Combo Elements
+
+These elements are designed to create easy-to-use combinations of other smaller components. Ideally they also bundle up attributes into a few configurations.
+
+**Example: [Hide-show](https://github.com/patternfly/patternfly-elements/blob/master/elements/pfe-hide-show)**
+
+
+## @todo should features be opt-in or opt-out?
+- If it's both, that’s confusing
+- If they are opt-in, it ensures that components don't change themselves on people's sites who are asking for all minor versions of components.
+
+
+
+
+# Attribute naming 
+
+## General
+- Attribute names should be specific so they are easy to read and understand. 
+- Use standard [HTML global attributes](https://www.w3schools.com/tags/ref_standardattributes.asp) and / or proper [ARIA attributes](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA) when appropriate
+    - Example: `<pfe-pagination hidden>` `<pfe-cta title="RHEL features">`
+
+## Prefixing attributes
+
+- To avoid collisions with other libraries, known and unknown as well as new HTML tags and attributes
+- To ensure folks who have no prefixing on their existing sites can use PF Elements without worry of collisiding CSS or JavaScript
+- To  have a single-source-of-truth, i.e. one class or attribute for styling vs. chaining attributes
+
+## Optional component features should be activated with a unique attribute instead of values within a generic attribute
+- The pagination component has opt-in features like enbaling the previous and next buttons. Ideally items are treated like booleans and can be toggled on if the attribute name is present on the tag. However if a feature requires a specific value to be passed in, such as a a number of results shown per page, it is fine to look for a value.
+- Example: 
+    - YES: `<pfe-pagination pfe-prev-next pfe-jump-nav pfe-results="10">`
+    - NO: `<pfe-pagination pfe-features="prev next caret">`
+
+
+## Example attributes with values
+Some attributes are standardized. 
+
+- `pfe-type="content"`
+    - Intended to provide context about parent child relationships and separation of concerns
+    - Other values: `container combo`
+- `pfe-orientation="vertical"`
+    - To change alignment of a container component 
+    - Other values: `horizontal`
+- `pfe-state="expanded"`
+    - To reflect the desired state of a component on render. It may be toggled.
+    - Other values: `collapsed`
+- `pfe-variant="earth"`
+    - The purpose is to group various styles together into one batch of settings, and to be abstract enough to allow for the evolution of the design system. The values are arbirtary and do not convey information about the styles.
+    - Other values: earth, wind, fire, water
+- `pfe-surface-color="complement"`
+    - Purpose: Invoke palette color overrides
+    - Example values: `base accent`
+    - @todo Can this mean background color and/or background color?
```

Comment:
Suggest adding the following to the bottom of the doc:

> ## Variants
> A variant is an attribute (`pfe-variant`) used across all our components which allows for sets of styles to be grouped together into one batch setting.  These styles are connected in the design system and allows for one pattern to communicate effectively in different design situations.  For example, the tab component's default look and feel resembles the manilla folder design with square boxes around each tab and no line below the active tab.  Other visual treatments of this same pattern might include one thick solid underline beneath or beside the tabs or making each tab resemble a button.  
> - `pfe-variant="fire"`
> - Other values: `earth`, `wind`, `water`

Reply:
More awesome suggestions, thanks! Updated